README.md

```diff
@@ -66,30 +66,30 @@ git reset --hard
 
 ---
 
-- To work on the GitHub workflows that perform the automated, scheduled pre-rendering:
-  - .github/
+## Local Development
```

Comment:
This commit (289edb62496b4fbaf3d7bed445680f8fbb19789d) is inconsistent. It removes some description of directories that should have been part of 5481388b99ee2d34909e2c5ab42b872965f106e9. This reduces the clarity of this PR and makes it quite hard to review.